gunicorn/app/wsgiapp.py

```diff
@@ -13,22 +13,21 @@
 class WSGIApplication(Application):
     def init(self, parser, opts, args):
         if opts.paste:
-            app_name = 'main'
-            path = opts.paste
-            if '#' in path:
-                path, app_name = path.split('#')
-            path = os.path.abspath(os.path.normpath(
-                os.path.join(util.getcwd(), path)))
+            from .pasterapp import has_logging_config
 
-            if not os.path.exists(path):
-                raise ConfigError("%r not found" % path)
+            config_uri = os.path.abspath(opts.paste)
+            config_file = config_uri.split('#')[0]
 
-            # paste application, load the config
-            self.cfgurl = 'config:%s#%s' % (path, app_name)
-            self.relpath = os.path.dirname(path)
+            if not os.path.exists(config_file):
+                raise ConfigError("%r not found" % config_file)
```

Comment:
Should this be `parser.error()` or `parser.exit()`?

Reply:
do you want to change this error?

Reply:
Maybe it's good to show usage here.

How do you feel if we do `parser.error("%r not found" % opts.paste)`?

Reply:
hrm but the issue is that it wouldn't be catch by the `cli`, would it?